packages/core/src/controllers/subscriber.ts

```diff
@@ -297,14 +296,12 @@ export class Subscriber extends ISubscriber {
   }
 
   private async resubscribe(subscription: SubscriberTypes.Active) {
-    const { topic, relay } = subscription;
-    const params = { topic, relay };
-    this.pending.set(params.topic, params);
-    const id = await this.rpcSubscribe(params.topic, params.relay);
-    this.onResubscribe(id, params);
-    if (this.ids.includes(subscription.id)) {
-      const reason = ERROR.RESUBSCRIBED.format({ topic: subscription.topic });
-      this.deleteSubscription(subscription.id, reason);
+    if (!this.ids.includes(subscription.id)) {
```

Comment:
Could you outline what changed here, just simplified logic by inverting the if condition?

Reply:
Essentially, before it subscribed, then checked if subscription.id is already included in subscription, if it was it then deleted subscription. I changed it so it checks if subscription.id is not already included and only if it is not, subscribes avoiding the need to delete / cleanup afterwards